app/src/main/java/com/github/HumanLearning2021/HumanLearningApp/MainActivity.kt

```diff
@@ -1,11 +1,16 @@
 package com.github.HumanLearning2021.HumanLearningApp
 
+import android.content.Intent
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
 
 class MainActivity : AppCompatActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        setContentView(R.layout.activity_main)
+        setContentView(R.layout.activity_dummy_categorized_picture_test)
```

Comment:
A button which opens DummyCategorizedPictureTestActivity would make demoing the app easier.